rs/replicated_state/src/canister_state/execution_state.rs

```diff
@@ -516,11 +521,35 @@ impl PartialEq for ExecutionState {
             metadata,
             last_executed_round,
             next_scheduled_method,
+            is_wasm64,
         )
     }
 }
 
 impl ExecutionState {
+    pub fn new_for_testing(
```

Comment:
Optional simplification: Since we have a `new_for_testing` function now, we could skip some of the arguments here and generate a default value in the function body. For some of these fields all the tests set them to some arbitrary value anyway. It definitely applies to `canister_root`, but might also apply to some of the other fields.

Reply:
Thanks, I treated this together with Adam's comment.